.github/workflows/CI.yml

```diff
@@ -30,7 +42,12 @@ jobs:
       - run: |
           julia --project --code-coverage=user -e '
             using Pkg
-            Pkg.GitTools.clone(Pkg.Types.Context(), "https://github.com/JuliaIO/JLDArchives.jl.git", "JLDArchives")
+            url = "https://github.com/JuliaIO/JLDArchives.jl.git"
```

Comment:
I think all of the quotes here need escaping.

Reply:
I was just going to ask if it was a quoting issue on Windows.